ansible/ansible-ai-connect-service
# AAP-33632: ModelPipelines: Implement invoke(..) method

Jira Issue: https://issues.redhat.com/browse/AAP-33632

## Description
Refactoring away of explicit `infer(..)`, `codematch(..)`, `generate_playbook(..)` and `explain_playbook(..)` methods.

A single `invoke(..)` method exists on a given _pipeline_. The parameters for each _feature_ differ.

## Testing
N/A Refactoring exercise.

### Steps to test
1. Pull down the PR
2. Run Wisdom Service locally
3. Connect with VSCode
4. Check things stil work

### Scenarios tested
As above

## Production deployment
